style.css

```diff
@@ -24,8 +22,9 @@ body {
   background-color: hsla(0, 100%, 10%, 0.3);
   top: 5rem;
   left: 10rem;
-  height: 200px;
-  width: 400px;
+  width: max-content;
```

Comment:
Can you share a screenshot of what it looks in your broswer ?

Reply:
It looks the same except the space needed is reduced. Plus to make it responsive I have to change some layouts in mobile

![Screenshot from 2023-01-29 09-25-26](https://user-images.githubusercontent.com/108436933/215309153-b6c13a84-e88c-4615-8e34-5cd92a5a4560.png)